tests/test.rfb.js

```diff
@@ -89,7 +89,7 @@ describe('Remote Frame Buffer Protocol Client', function () {
     after(FakeWebSocket.restore);
 
     before(function () {
-        this.clock = clock = sinon.useFakeTimers();
+        this.clock = clock = sinon.useFakeTimers(Date.now());
```

Comment:
Why do we need `Date.now()`? It introduces a variability in the tests, which generally is unwanted.

Reply:
We don't need it on this level - that was included by mistake.